modules/git/blame.go

```diff
@@ -134,5 +156,35 @@ func CreateBlameReader(ctx context.Context, repoPath, commitID, file string) (*B
 		reader:         reader,
 		bufferedReader: bufferedReader,
 		done:           done,
+		ignoreRevsFile: ignoreRevsFile,
 	}, nil
 }
+
+func tryCreateBlameIgnoreRevsFile(commit *Commit) *os.File {
+	entry, err := commit.GetTreeEntryByPath(".git-blame-ignore-revs")
+	if err != nil {
+		return nil
+	}
+
+	r, err := entry.Blob().DataAsync()
+	if err != nil {
+		return nil
+	}
+	defer r.Close()
+
+	f, err := os.CreateTemp("", "gitea_git-blame-ignore-revs")
```

Comment:
I think your approach will fail if two blames are requested at the same time, correct?
One will overwrite the other, if I see that correctly.

Reply:
`os.CreateTemp` is designed to create different files in this case.

Reply:
Ah, I looked at the wrong `CreateTemp` documentation.